### ISSUE TITLE
dataset issues

### ISSUE THREAD

**Issue by Wateryhq** [uma-pi1/kge#279]
1."Can it train in Chinese?"
2."I still don't understand how to create the dataset.""I only understand the training set, validation set, and test set. I didn't understand the rest."

**Comment by rgemulla** [comment]
See here under key "dataset" for a description of dataset format and how it is used: https://github.com/uma-pi1/kge/blob/master/kge/config-default.yaml.

**Comment by Wateryhq** [comment]
看了还是有不明白的地方。I’ve read it, but there’s still something I don’t understand.

**Comment by Wateryhq** [comment]
感谢您的来信！
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;杨华清
thank you for your mail！
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;YangHuaqing




752936051
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "uma-pi1/kge"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年10月9日(星期一) 下午3:19
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [uma-pi1/kge] dataset issues (Issue #279)





 
See here under key "dataset" for a description of dataset format and how it is used: https://github.com/uma-pi1/kge/blob/master/kge/config-default.yaml.
